<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
}

QWidget{

}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="robo_mp_einlesen_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(85, 87, 83);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
	color:rgb(0, 0, 0);
    min-width: 5em;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Roboter Messpunkte einlesen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
background-color: rgb(186, 189, 182);
color
	color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
}</string>
    </property>
    <property name="text">
     <string>/home/timohin/Documents</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}
</string>
    </property>
    <property name="text">
     <string>Pfad zum csv-Ordner (config,messpunkte,trafomatrix)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_x">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_y">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_z">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

}</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_x_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_y_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_z_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
background-color: rgb(186, 189, 182);
color
	color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>Anzahl der erfassten Messpunkte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trafom_berechnen_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(85, 87, 83);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
	color:rgb(0, 0, 0);
    min-width: 5em;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Trafomatrix berechnen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(85, 87, 83);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
	color:rgb(0, 0, 0);
    min-width: 5em;
    padding: 6px;

}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
background-color: rgb(186, 189, 182);
color
	color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(85, 87, 83);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
	color:rgb(0, 0, 0);
    min-width: 5em;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>Statusmeldung:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trafom_einlesen_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(85, 87, 83);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
	color:rgb(0, 0, 0);
    min-width: 5em;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Trafomatrix von csv einlesen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prog_beenden_btn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>470</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(85, 87, 83);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: rgb(0, 0, 0);
	font: 75 14pt &quot;Waree&quot;;
	color:rgb(0, 0, 0);
    min-width: 5em;
    padding: 6px;

}</string>
    </property>
    <property name="text">
     <string>Programm Beenden</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Roboter-Koordinaten</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracking-Koordinaten</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(204, 0, 0);
font: 75 14pt &quot;Waree&quot;;

}</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slot_calc()</slot>
 </slots>
</ui>
